docs/modules/ROOT/partials/getting-started/con-registry-client.adoc

```diff
@@ -3,6 +3,18 @@
 [id="registry-client-intro"]
 = {registry} Java client overview
 
-You can manage artifacts stored in {registry} using a Java client application. You can create, read, update, or delete artifacts stored in the registry using the {registry} Java client classes...  
+You can manage artifacts stored in {registry} using a Java client application. You can create, read, update, or delete artifacts stored in the registry using the {registry} Java client classes.
 
-More TBD
+You can access the {registry} Java client by adding the proper dependency to your project, see xref:writing-registry-client[].
+
+The {registry} client is autocloseable and is implemented using Retrofit and OkHttp. This gives the user the possibility of customize its usage like, for example, adding custom headers or enabling TLS auth support.
+
+The TLS auth support can be enabled and configured using the following properties:
+
+* apicurio.registry.request.ssl.truststore.location
+* apicurio.registry.request.ssl.truststore.password
+* apicurio.registry.request.ssl.truststore.type
+* apicurio.registry.request.ssl.keystore.location
```

Comment:
Suggest providing an example of configuring TLS in client code.

Reply:
Since we're mentioning the properties needed, and they are pretty common, I'm not sure about adding an example here. @EricWittmann wdyt?

Reply:
I suggest we add that as a separate issue in GitHub.  I'm in favor of an example, but I'm not sure the docs are the appropriate forum for that.  Perhaps the apicurio-registry-examples repository instead.  I'd prefer a working (practical) example.

Reply:
[Issue](https://github.com/Apicurio/apicurio-registry-examples/issues/4) created.

Reply:
Thanks guys, yes demo is even better :)